Libs/MRML/Logic/vtkMRMLSliceLogic.cxx

```diff
@@ -549,11 +549,23 @@ void vtkMRMLSliceLogic::ProcessMRMLLogicsEvents()
     plane->SetOrigin(outPt3);
 
```

Comment:
Thanks for the review

> unused min and max macros at the top of this file could be removed

Done. Integrated in [r26286](http://viewvc.slicer.org/viewvc.cgi/Slicer4?view=revision&revision=26286)

> could be cleaner using std::max

Done. Integrated in [r26287](http://viewvc.slicer.org/viewvc.cgi/Slicer4?view=revision&revision=26287)